src/main/resources/init.sql

```diff
@@ -0,0 +1,102 @@
+-- user
+DROP TABLE IF EXISTS `kotlin.backend.db`.`user`;
```

Comment:
Let's change the name to simply `mentor_portal_db`

And if you add a `USE mentor_portal_db` at the very top of this script, you can remove the reference to the db on the rest of the scripts here.